### DISCUSSION TITLE
is posible to reverse search?

### DISCUSSION THREAD

**Discussion by kucingkembar** [stashapp/stash#5230] [Q&A]
sorry for my bad English, and newbie question
I know the actress names, the studio name, and the title of a video,
but any "end-points" produce "No scenes found",
is possible to ask video/scene other details using the actress name + studio name + title, 
instead perceptual-hashes?

**Comment by feederbox826** [comment]
put those in the search box in tagger view, it's not as reliable as phash but might get you the result you want - title is your best bet

**Reply by kucingkembar** [reply]
thank you
